container/push.bzl

```diff
@@ -58,6 +58,13 @@ def _impl(ctx):
     config_arg = "--config=%s" % _get_runfile_path(ctx, image["config"])
     manifest_arg = "--manifest=%s" % _get_runfile_path(ctx, image["manifest"])
 
+    optional_args = ""
+    # If the docker toolchain is configured to use a custom client config
+    # directory, use that instead
+    toolchain_info = ctx.toolchains["@io_bazel_rules_docker//toolchains/docker:toolchain_type"].info
+    if toolchain_info.client_config != "":
```

Comment:
once its not mandatory this can be if not toolchain_info.client_config

Reply:
Keeping this one as we discussed